### ISSUE TITLE
What is this behaviour

### ISSUE THREAD

**Issue by karma1212** [cthos/gw2-api#25]
```
function exchange (quantity){
api.callAPI('/commerce/exchange/coins', { 'quantity': 1150000}, false).then(function (res){
	        console.log(res);
	        var getGems = res.quantity;
	        console.log(res.coins_per_gem*400/10000);
               }
}
```
exchange (1149999) returned a quantity of 359
exchange (1150000) returned a quantity of 409 
exchange (1150001) returned a quantity of 356

Seems not logic to jump this way

> { coins_per_gem: 3203, quantity: 359 }
> 128.12
> { coins_per_gem: 2811, quantity: 409 }
> 112.44
> { coins_per_gem: 3230, quantity: 356 }
> 129.2

**Comment by cthos** [comment]
Wowzers, I missed this issue for a good long time.

To answer the question, that's directly calling the exchange API here:
https://wiki.guildwars2.com/wiki/API:2/commerce/exchange/gems

Which can return different numbers depending on the current exchange rate (it fluctuates), and the number of coins you're willing to exchange. 

tl;dr - not sure why the exchange API fluctuates wildly when you change the quantity at high numbers. There's a forum post asking the same question:
https://en-forum.guildwars2.com/discussion/44071/api-v2-commerce-exchange-gems-with-different-quantities

But doesn't look like the ANet devs got back to them.
